### DISCUSSION TITLE
How to change the default font of Wiki.js?

### DISCUSSION THREAD

**Discussion by mhasgari** [requarks/wiki#3938] [Help / Questions]
Hi
I want to change de default font of Wiki.js in Farsi language. I tried in so many ways, but none were successful!
How can I do that?

**Comment by mhasgari** [comment]
> Hi I want to change de default font of Wiki.js in Farsi language. I tried in so many ways, but none were successful! How can I do that?

How would it be to change wiki.js's default font?

**Comment by switchupcb** [comment]
Requesting an update.

**Comment by ph0** [comment]
It is a mystery.
